models/edr/data_monitoring/anomaly_detection/metrics_anomaly_score.sql

```diff
@@ -70,7 +70,7 @@ final as (
 
     select *,
         case
-            when abs(z_score) > {{ elementary.get_config_var('anomaly_score_threshold') }} then true
+            when abs(z_score) > {{ elementary.get_config_var('anomaly_sensitivity') }} then true
```

Comment:
Since we are changing the config var name, I've needed to update the name in a previously untouched file